www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+
+Run the following command in your terminal to create a new directory called `svelte-snowpack` with the minimal template installed:
+
+```bash
+npx create-snowpack-app svelte-snowpack --template @snowpack/app-template-minimal
+```
+
+Head to the new `svelte-snowpack` directory and start Snowpack with the following two commands:
+
+```bash
+cd svelte-snowpack
+npm run start
+```
+
+You should see your new website up and running!
+
+<div class="frame"><img src="/img/guides/react/minimalist-hello-world.png" alt="screenshot of project-template-minimal, which shows 'Hello world' in text on a white background." class="screenshot"/></div>
+
+Now that you have a basic project up and running! The next step is to install Svelte. Run the following command in your project directory:
+
+```bash
+npm install svelte --save
+```
+
+> 💡 Tip: add the "--use-yarn" or "--use-pnpm" flag to use something other than npm
+
+Snowpack [plugins](/plugins) are a way to extend Snowpack's capabilities without having to do custom configuration yourself. Install the `@snowpack/plugin-svelte` plugin so that Snowpack knows how built `.svelte` files into JavaScript and CSS files that run in the browser:
+
+```bash
+npm install @snowpack/plugin-svelte --save-dev
+```
+
+Once installed, you'll need to add the plugin to your Snowpack configuration file (`snowpack.config.js`) so that Snowpack knows to use it:
+
+```diff
+// snowpack.config.js
+
+module.exports = {
+  mount: {
+    /* ... */
+  },
+  plugins: [
+-    /* ... */
++    '@snowpack/plugin-svelte'
+  ],
+```
+
+Restart your Snowpack dev server to run it with the new configuration. Exit the process (ctrl + c in most Windows/Linux and cmd + c for Mac) and start it again with `npm run start`.
+
+> 💡 Tip: Restart the Snowpack development server when you make configuration changes (changes to the `snowpack.config.js`).
+
+The Snowpack Development server will show the following as it recognizes your new dependency (Svelte) and turns it into ESM:
+
+```bash
+[snowpack] installing dependencies...
+[snowpack] ✔ install complete! [0.45s]
+[snowpack]
+  ⦿ web_modules/                                size       gzip       brotli
+    ├─ svelte-hmr/runtime/hot-api-esm.js        22.17 KB   7.42 KB    6.3 KB
+    ├─ svelte-hmr/runtime/proxy-adapter-dom.js  5.17 KB    1.65 KB    1.38 KB
+    └─ svelte/internal.js                       52.78 KB   13.24 KB   11.45 KB
+```
+
+## Create your first Svelte component
+
+With `@snowpack/plugin-svelte` installed, Snowpack will build `.svelte` files to the HTML, JS, and CSS needed to run in the browser. Create your first `.svelte` component named `App.svelte` with the following code:
```

Comment:
Hard to tell if this first sentence is summarizing where we are so far, or explaining what you're about to do. Maybe instead:

 > You now have your Snowpack environment set up to build `.svelte` files for the browser. Now it's time to create your first Svelte component file!

> Create a file named `App.svelte` in your project directory with the following code: